pyproject.toml

```diff
@@ -46,6 +46,7 @@ dev = [
   "check-manifest",
   "types-PyYAML",
   "types-requests",
+  "ffmpeg-python"
```

Comment:
This dependency should be added with the required dependency, not with the optional ones as is here.